File: C/coordenadas.c
#include <stdio.h>

int main() {

    double x, y;

    printf("Valor de X: \n");
    scanf("%lf", &x);
    scanf("%lf", &y);

    if (x > 0 && y > 0) {
        printf("Q1");
    }
    else if (x > 0  && y < 0) {
        printf("Q4");
    }
    else if (x < 0 && y < 0) {
        printf("Q3");
    }
    else if (x < 0 && y > 0) {
        printf("Q2");
    }
    else if (x == 0 && y == 0) {
        printf("Origem");
    }
    else if (x == 0) {
        printf("Eixo Y");
    }
    else {
        printf("Eixo X");
    }

    return 0;
}

File: Java/Coordenadas.java
import java.util.Locale;
import java.util.Scanner;

public class Coordenadas {
    public static void main(String[] args) {
        
        Locale.setDefault(Locale.US);
        Scanner sc = new Scanner(System.in);

        double x, y;

        System.out.print("Valor de X: ");
        x = sc.nextDouble();
        System.out.print("Valor de Y: ");
        y = sc.nextDouble();

        if (x > 0 && y > 0) {
            System.out.println("Q1");
        }
        else if (x > 0  && y < 0) {
            System.out.println("Q4");
        }
        else if (x < 0 && y < 0) {
            System.out.println("Q3");
        }
        else if (x < 0 && y > 0) {
            System.out.println("Q2");
        }
        else if (x == 0 && y == 0) {
            System.out.println("Origem");
        }
        else if (x == 0) {
            System.out.println("Eixo Y");
        }
        else {
            System.out.println("Eixo X");
        }

        sc.close();
    }
}